source/postprocess/gravity_point_values.cc

```diff
@@ -409,13 +409,13 @@ namespace aspect
                      << position_satellite[0] << ' '
                      << position_satellite[1] << ' '
                      << position_satellite[2] << ' '
-                     << std::setprecision(18) << g << ' '
-                     << std::setprecision(18) << g.norm() << ' '
-                     << std::setprecision(18) << g_theory << ' '
-                     << std::setprecision(9) << g_potential << ' '
-                     << std::setprecision(9) << g_potential_theory << ' '
-                     << std::setprecision(9) << g_anomaly << ' '
-                     << std::setprecision(9) << g_anomaly.norm() << ' '
+                     << std::setprecision(precision) << g << ' '
+                     << std::setprecision(precision) << g.norm() << ' '
+                     << std::setprecision(precision) << g_theory << ' '
+                     << std::setprecision(precision) << g_potential << ' '
+                     << std::setprecision(precision) << g_potential_theory << ' '
+                     << std::setprecision(precision) << g_anomaly << ' '
+                     << std::setprecision(precision) << g_anomaly.norm() << ' '
                      << g_gradient[0][0] *1e9 << ' '
```

Comment:
gradients below will have the same precision as stated in line 418. In fact, since the precision will now be the same for every quantities, should I just not delete the std::setprecision at lines 413-418 and just keep one? Or should I just leave it like this?

Reply:
@LudovicJnnt - I'm not sure I fully understand the question. Is the precision for this file now set globally above? If not, is the standard out precision sufficient? Sorry, I'm likely overlooking something from the previous PR.

Reply:
@naliboff , sorry for the confusion, I hope this is better explained:
when specified `std::precision(N) <<  quantities_1 << quantity_2 << ... ` all quantities (1, 2, ...) following have the same precision N.
This is true until a new specification std::precision(M) is stated:
`std::precision(N) <<  quantities_1 << quantity_2 << ... << std::precision(M) << quantity_3`

In my case, all the gradient quantities at lines 419 to 430 will have the precision stated at line 418. But in the previous lines, I basically reset the precision (for the same precision value) which is therefore completely pointless. I should only set the precision once (at line 413)

Now my question is if I include the same precision for gravity and potential also for gradients, or I set a new precision value just for the gradient fixed to 9 only.

Reply:
@LudovicJnnt - Gotcha, thanks for the explanation. The only reason to reset the precision for every value in lines 419-430 is if you think it will be useful down the line to specify a different precision for each value. If not, I would agree that is best to just set the precision once at line 413.

> Now my question is if I include the same precision for gravity and potential also for gradients, or I set a new precision value just for the gradient fixed to 9 only.

To avoid confusion I would use the same precision for the gradients, but perhaps add a comment that the high precision is really only needed for the gravity and potential values.